File: django_backend/django_backend/restauth/urls.py
from django.conf.urls import url

from .views import *

urlpatterns = [
    #Does not require quthentication
    url(r'^login/$', LoginView.as_view(), name=r"login"), # post: username, password
    url(r'^registration/$', SignUpView.as_view(), name=r"signup"), # post: username, email, password1, password2
    url(r'^registration/verify-email/$', ValidateEmailView.as_view(), name=r"verify_email"), # get: with params ?token={token}
    url(r'^password/reset/$', ResetPasswordView.as_view(), name=r"pwd_reset"), # post: email
    url(r'^password/reset/confirm/$', ConfirmPasswordView.as_view(), name=r"pwd_confirm"), # post: new_password1, new_password2, token
    
    # require authentication
    url(r'^logout/$', LogoutView.as_view(), name=r"logout"), # get + authentication header
    url(r'^password/change/$', PasswordChangeView.as_view(), name=r"pwd_change"), # post: old_password, new_password1, new_password2
    url(r'^user/$', UserView.as_view(), name=r"user_profile"), # post+get
]

File: react_frontend/src/constants/urls.js
const ROOT_URL = "http://localhost:8000/";

export const AuthUrls = {
    LOGIN: `${ROOT_URL}rest-auth/login/`,
    LOGOUT: `${ROOT_URL}rest-auth/logout/`,
    SIGNUP: `${ROOT_URL}rest-auth/registration/`,
    CHANGE_PASSWORD: `${ROOT_URL}rest-auth/password/change/`,
    RESET_PASSWORD: `${ROOT_URL}rest-auth/password/reset/`,
    RESET_PASSWORD_CONFIRM: `${ROOT_URL}rest-auth/password/reset/confirm/`,
    USER_ACTIVATION: `${ROOT_URL}rest-auth/registration/verify-email/`,
    USER_PROFILE: `${ROOT_URL}rest-auth/user/`,
};

export const ServiceUrls = {
    SHOPS: `${ROOT_URL}shops/`,
    FAVORITE_SHOPS: `${ROOT_URL}shops/favorite/`,
    DISLIKE_SHOP: `${ROOT_URL}shops/dislike/`,
};